.github/workflows/e2e.yaml

```diff
@@ -22,6 +29,11 @@ jobs:
       with:
         go-version: "1.14"
 
+    - name: Reclaim some disk space
+      run: |
+        sudo apt-get clean
+        df -h
```

Comment:
Don't think this is required.

Reply:
yeah, maybe it's not. e2e test is still failing time to time even with this explicit clean-up step, so maybe it's unrelated

Reply:
you are right, there is no any visible effect of this command, removed